static/js/app.js

```diff
@@ -35,3 +35,31 @@ allLeaders.forEach(function(orgList) {
     orgList.appendChild(orgList.children[(Math.random() * i) | 0])
   }
 })
+
+document.getElementById('progress').textContent = gciProgress()
+
+function gciProgress() {
+  const competitionOpen = new Date(
+    document.getElementById('progress').dataset.competitionopen
```

Comment:
Storing the result of getElementById would be better instead of doing it again and again.